src/index.njk

```diff
@@ -112,17 +109,42 @@ is_generic: true
         </div>
         <p>{{ lastEpisode.data.youtube_id | youtubeLink }}</p>
       </div>
+    </div> #}
+
+    <div class="my-8">
+      <h2 class="text-gray-900 dark:text-gray-100 text-xl lg:text-2xl font-bold mb-4">Latest episodes</h2>
+      <div class="bg-white dark:bg-slate-800 rounded-sm overflow-hidden shadow-sm prose-slate dark:prose-dark">
+        <a href="{{lastEpisode.url}}" class="text-lg text-gray-900 dark:text-gray-100 hover:text-primary-500 dark:hover:text-secondary-400">
+          <div class="grid md:grid-cols-2 xs:grid-rows-2">
+            <div class="aspect-[16/9] bg-gray-200 dark:bg-slate-700"
+                  style="background-image: url('{{ lastEpisode.data.youtube_id | youtubePreview(lastEpisode.url) -}}');
+                  background-size: cover; background-position: center;"></div>
+            <div>
+              <h3 class="p-4 font-bold">
+                {{ lastEpisode.data.episode }}. {{lastEpisode.data.title}}
+              </h3>
+              <h6 class="p-4">Published {{ lastEpisode.data.publish_date | formatDate }}</h6>
+            </div>
+          </div>
+        </a>
+      </div>
     </div>
 
-    <div class="my-8 prose prose-sm md:prose-lg lg:prose-xl dark:prose-dark">
-      <h2 class="text-gray-900 dark:text-gray-100">Previous episodes</h2>
-      <ul class="">
+    <div class="my-8">
+      <div class="grid grid-cols-1 sm:grid-cols-2 md:grid-cols-3 lg:grid-cols-4 gap-8">
```

Comment:
semantically it might still make sense to use a `<ul>` as the container here as we are still representing a list. And actually, since this is ordered by episode number it might even make more sense to be an `<ol>`...

This is a very debatable opinion, though. Here are more references that can be useful if you are curious: https://stackoverflow.com/questions/19307443/semantic-html-of-an-articles-list